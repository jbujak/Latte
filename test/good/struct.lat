class c {
	int n;
	string s;
}
int main() {
	c obj = new c;
	obj.n = 1;
	obj.s = "asdf";

	printInt(obj.n);
	printString(obj.s);

	change(obj);

	printInt(obj.n);
	printString(obj.s);

	return 0;
}

void change(c arg) {
	arg.n = arg.n + 1;
	arg.s = "qwer";
}
