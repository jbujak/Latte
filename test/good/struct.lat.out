1
asdf
2
qwer
50
100
